### PR TITLE
MARBLE-1197 Fix footer sx

### PR DESCRIPTION
**Pull request by jmille79**

### CODE REVIEW

**Comment by jmille79** [comment]
Screen Shot 2020-10-19 at 3 02 08 PM

File: site/src/@ndlib/gatsby-theme-marble/components/Layout/PageWrapper/Footer/index.js [hunk 1]
```diff
@@ -14,66 +14,62 @@ export const Footer = () => {
...
+            href='https://sniteartmuseum.nd.edu/'
+            sx={sx.imageWrapper}
+          >
+            <img
+              src={sniteLogo}
+              sx={sx.image}
+              alt='Snite Art Museum'
+            />
+          </a>
+          <p className='contact-info'>
```
Comment (danthewolfe) [inline site/src/@ndlib/gatsby-theme-marble/components/Layout/PageWrapper/Footer/index.js:35]: I've been getting the following warning: 
```
index.js:2177 Warning: validateDOMNesting(...): <div> cannot appear as a descendant of <p>.
    in div (at Footer/index.js:37)
    in p (at Footer/index.js:36)
```
So can we get the `<p/>`s here changed to `<div/>` or <span/>`?

File: site/src/@ndlib/gatsby-theme-marble/components/Layout/PageWrapper/Footer/sx.js [hunk 2]
```diff
@@ -23,23 +24,26 @@ module.exports = {
...
   },
-  copyright: {
-    padding: '10px',
-    left: '10px',
-  },
+  footerRightColumn: {
+    textAlign: 1,
+    right: ['auto', '0', '0'],
+    align: 'right',
+  }
```
[the remark this answers is not in the document]
Comment (danthewolfe) [reply site/src/@ndlib/gatsby-theme-marble/components/Layout/PageWrapper/Footer/sx.js:48]: Add trailing comma to line 48 to make hound happy.

File: site/src/@ndlib/gatsby-theme-marble/components/Layout/PageWrapper/Footer/index.js [hunk 3]
```diff
@@ -14,66 +14,62 @@ export const Footer = () => {
   return (
```
Comment (danthewolfe) [inline site/src/@ndlib/gatsby-theme-marble/components/Layout/PageWrapper/Footer/index.js:14]: The line before code review starts contains an unused variable. Can we get this line removed?
```
  const date = new Date()
```

**Review by danthewolfe — CHANGES_REQUESTED** [review]
Visually, looks much better. See comments for a few additional changes.

File: site/src/@ndlib/gatsby-theme-marble/components/Layout/PageWrapper/Footer/index.js (hunk 1 above)
Comment (danthewolfe) [reply site/src/@ndlib/gatsby-theme-marble/components/Layout/PageWrapper/Footer/index.js:34]: Looks like you got the other instances of `<div/>` as a child of `<p/>`, get line 34 and line 41, then 👍 .

**Review by danthewolfe — APPROVED** [review]
See latest comment on one more `<p/>`, then 👍 .

**Review by mlkirkpatrick — APPROVED** [review]

**Review by SteveMattison — APPROVED** [review]

**Pull request merged**